zerver/webhooks/slack/doc.html

```diff
@@ -0,0 +1,21 @@
+<p>Get notifications from Slack for messages on your team's public channels</p>
+<p>First, create your desired stream to receive notifications from slack</p>
+
+
+<p>
+   Next go to the following url <code>https://yourteamname.slack.com/apps/build</code>
+</p><br/>
+
+<p>
+    Select <code> Make a custom integration </code> option <br/><br/>
+    <img class="screenshot" src="/static/images/integrations/slack/001.png"/><br/><br/>
+    Next choose the <code> Outgoing WebHooks</code> option<br/><br/>
+    <img class="screenshot" src="/static/images/integrations/slack/002.png"/><br/><br/>
+    Now under <code> Integration Settings </code> fill in the channel you'd like to get your notifications from<br/><br/>
+    Then fill <code>{{ external_api_uri_subdomain }}/v1/external/slack?api_key=abcdefgh&amp;stream=slack</code> as your URL<br/><br/>
+    <img class="screenshot" src="/static/images/integrations/slack/003.png"/><br/><br/>
+    At last, Save your settings<br/><br/>
```

Comment:
Here are fixes to the grammar and punctuation here:
~~~
    Select the <code> Make a custom integration </code> option. <br/><br/>
    <img class="screenshot" src="/static/images/integrations/slack/001.png"/><br/><br/>
    Next, choose the <code> Outgoing WebHooks</code> option.<br/><br/>
    <img class="screenshot" src="/static/images/integrations/slack/002.png"/><br/><br/>
    Now, under <code> Integration Settings </code>, fill in the channel you'd like to get your notifications from.<br/><br/>
    Then fill <code>{{ external_api_uri_subdomain }}/v1/external/slack?api_key=abcdefgh&amp;stream=slack</code> as your URL<br/><br/>
    <img class="screenshot" src="/static/images/integrations/slack/003.png"/><br/><br/>
    Finally, save your settings.<br/><br/>
~~~